app/scripts/modules/core/pipeline/config/triggers/cron/cronTrigger.html

```diff
@@ -27,4 +27,19 @@
       {{vm.validationMessages.error}}
     </div>
   </div>
+
+  <div class="form-group" ng-if="vm.fiatEnabled">
```

Comment:
It might be worth refactoring this template into a component (it could be a controller-less component).